posts/2022-02-21-rust-analyzer-joins-rust-org.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "rust-analyzer joins the Rust organization!"
+author: The rust-analyzer Team on behalf of the entire Rust Team
+---
+
+We have an exciting announcement to make!
+The [rust-analyzer](https://rust-analyzer.github.io) project, a new implementation of the Language Server Protocol (LSP) for Rust, is now officially a part of the wider Rust organization! :tada:
+
+We want to start by thanking everyone who has gotten us this far from contributors, to sponsors, to all the users of rust-analyzer in the Rust community. We could not have done this without you.
+
+The immediate impact of this organizational change is limited -- nothing changes for rust-analyzer users or contributors.
+However, this change unblocks technical work to make rust-analyzer the officially recommended language server for Rust in the near future.
+
+If you were hesitant to try rust-analyzer before, today is a good opportunity to do so.
+Not only is it a very capable language server for Rust, but according to [VS Code statistics](https://marketplace.visualstudio.com/search?target=VSCode&category=Programming%20Languages&sortBy=Rating), it is one of the best rated LSP implementations *across* programming languages.
+We highly recommend giving rust-analyzer a spin today, even if it will take some more time for us to complete the due process and switch from the existing officially recommended LSP implementation (RLS) properly.
+
+rust-analyzer enjoys first-class support in many editors:
```

Comment:
```suggestion
rust-analyzer enjoys excellent support in many editors:
```

I feel like I typically see first-class support as 'built-in', rather than through extensions -- which I think in all editors is not currently the case?

Reply:
I was thinking in from the dual perspective -- rust-analyzer implements a bunch of LSP extensions, and those extensions are supported (via plugins) in most major editors. Ie, it's not about editor X having dedicated support for Rust, but rather rust-analyzer having dedicated support for editor X, if that makes sense. 

But yeah, the wording doesn't convey this, and it's not *that* imporant. What matters to the user is indeed that the support is excellent